src/components/Delegation/DelegationConfirmation.js

```diff
@@ -74,7 +75,7 @@ const messages = defineMessages({
     description: 'some desc',
   },
   delegationTxSignError: {
-    id: 'components.stakingcenter.delegationTxSignError',
```

Comment:
why are you changing this? It breaks all translations

Reply:
https://github.com/Emurgo/yoroi-mobile/blob/ea6ee0f74b34edf55ad9ef1081462a731cf54395/src/i18n/locales/en-US.json#L148